tests/scripts/localPathPVForOSD.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+function create_osd() {
+  lsblk
+  kubectl delete pv -l type=local-osd
+
+  if [ $device_type = lvm ] ; then
+    test_vg=rook-integration-test-vg
+    test_lv=rook-integration-test-lv
+    sudo pvcreate "${test_scratch_device}"
+    sudo vgcreate "${test_vg}" "${test_scratch_device}"
+    sudo lvcreate -l 100%FREE -n "${test_lv}" "${test_vg}"
+    test_device=/dev/"${test_vg}"/"${test_lv}"
+  fi
+
+  cat <<eof | kubectl apply -f -
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: local-osd
+  labels:
+    type: local-osd
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 9Gi
```

Comment:
Why 9G instead of 10G? Is there a rounding issue with the size of the disk in the CI?

Reply:
It's because CI environment only has 10GiB volume. I saw it in slack test channel.

https://rook-io.slack.com/archives/C764K425D/p1593105305031500
> Could you add another device to the CI? Right now we have a single 10G device, right? A second 10G device would be perfect.

If it's correct, creating VG and LV reduces the size of the volume that backs PVC. So I changed the size of PVC. If the volume size will be expanded to more than 10GiB, it's better.

CMIIR.

Reply:
@travisn Do you have any comment about this?

Reply:
That sounds good, no worries about using 9G